src/test/java/TestRunners/TestBrowser.java

```diff
@@ -107,11 +86,13 @@ public void testDesktopBrowserStatusCode() {
     public void testDesktopCustomDriverBrowser() {
         Configuration.automationType = DESKTOP_PLATFORM;
         Configuration.browser = "chrome";
+        Configuration.headless = true;
         open("https://www.google.com");
         stop();
         ChromeOptions options = new ChromeOptions();
         String userAgent = "Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)";
         options.addArguments("--user-agent=" + userAgent);
+        options.addArguments("--headless");
```

Comment:
Do you really need to provide both `Configuration.headless` and `--headless` Chrome option?

Reply:
The reason is because in this second time, that I start a driver, I do it in a custom way, creating the chrome driver and then setting it up for TestUI with `setDriver(chromeDriver);`